### PR TITLE
Update to new terms across all our code

### PR DESCRIPTION
**Pull request by terenc3t**
Here is the list of todos to update terms across all our code

- [x] Update main Sharding folder with the new terms
- [x] Update cli entry points with the new terms
- [x] Update validator_manager.sol with the new terms
- [x] Refactor client to collator client

### CODE REVIEW

File: sharding/client.go [hunk 1]
```diff
@@ -185,9 +185,9 @@ func (c *Client) Client() *ethclient.Client {
 	return c.client
 }
 
-// VMCCaller to interact with the validator management contract.
-func (c *Client) VMCCaller() *contracts.VMCCaller {
-	return &c.vmc.VMCCaller
+// VMCCaller to interact with the sharding manager contract.
+func (c *Client) VMCCaller() *contracts.SMCCaller {
```
Comment (prestonvanloon) [inline sharding/client.go]: SMCCaller

**Comment by prestonvanloon** [comment]
Please run tests and lint. Travis is red

**Comment by terenc3t** [comment]
Will do. Travis is red because VMC.sol and the bindings are still old

File: sharding/collator.go [hunk 2]
```diff
@@ -79,35 +79,35 @@ func checkShardsForProposal(c collatorClient, head *types.Header) error {
...
-func isAccountInValidatorSet(c collatorClient) (bool, error) {
+// The function calls IsCollatorDeposited from the SMC and returns true if
+// the client is in the collator pool
+func isAccountInCollatorPool(c collator) (bool, error) {
 	account := c.Account()
 
-	// Checks if our deposit has gone through according to the VMC
-	b, err := c.VMCCaller().IsValidatorDeposited(&bind.CallOpts{}, account.Address)
+	// Checks if our deposit has gone through according to the SMC
+	b, err := c.SMCCaller().IsCollatorDeposited(&bind.CallOpts{}, account.Address)
```
Comment (prestonvanloon) [inline sharding/collator.go]: Just return the result of this method call. No need for the following if else statement

File: sharding/collator.go [hunk 3]
```diff
@@ -79,35 +79,32 @@ func checkShardsForProposal(c collatorClient, head *types.Header) error {
...
+func isAccountInCollatorPool(c collator) (bool, error) {
 	account := c.Account()
 
-	// Checks if our deposit has gone through according to the VMC
-	b, err := c.VMCCaller().IsValidatorDeposited(&bind.CallOpts{}, account.Address)
-	if err != nil {
-		return false, err
-	}
+	// Checks if our deposit has gone through according to the SMC
+	b, err := c.SMCCaller().IsCollatorDeposited(&bind.CallOpts{}, account.Address)
```
Comment (prestonvanloon) [inline sharding/collator.go]: return c.SMCCaller()...

No need for these variables

**Comment by nisdas** [comment]
Rename from vmc.go to smc.go

**Pull request merged**